src/types/metaFactory.ts

```diff
@@ -47,7 +47,15 @@ export const MetaFactoryTypes: MetaFactoryTypes = {
       startingBlock: event.blockNumber,
       contractType: NFTContractTypeName.default,
       standard: NFTStandard.ERC721,
-      autoApprove: approved
+      autoApprove: approved,
+      typeMetadata: {
+        overrides: {
+          artist: {
+            artistId: event.args!.creator,
```

Comment:
ooh this and the mapper looks cool but maybe lets put it in a separate PR so we can get the basic nft indexing merged in first

Reply:
I think this has to included in this PR, because the type metadata needs to be set at the time of indexing. If we index the factories and then come back later with type overrides, we won't have access the the events anymore and won't be able to set this data unless we reprocess everything?